### DISCUSSION TITLE
Error on Windows 11

### DISCUSSION THREAD

**Discussion by victorpinhors** [valinet/ExplorerPatcher#1840] [General]
![image](https://github.com/valinet/ExplorerPatcher/assets/29187664/50e09fff-31b1-4d67-a5d1-eee685ff91e0)

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Comment by victorpinhors** [comment]
> Please post windows version and build. Please post explorer patcher version.

It's in the images sent... Version and Bild

![image](https://github.com/valinet/ExplorerPatcher/assets/29187664/6c8ff42a-a5ff-491b-a8b8-af6465709f81)

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. delete the registry keys: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher" "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by red-cap** [reply]
This has worked for me. thank you!

**Reply by Grin97** [reply]
This has worked for me. thank you!!! :)

**Reply by nebosite** [reply]
Worked for me!  Yeah!!!

**Reply by ascheel** [reply]
Worked for me until I rebooted (Thank you, Windows Update....)

Then it offered an update to *.56.1 and I took it, but still constant crashing.  Repeated the above steps and it's good to go.  BUT I haven't rebooted, so we'll see after the next reboot.

**Reply by victorpinhors** [reply]
Thks. resolved :-)

**Comment by victorpinhors** [comment]
> Try the following:
> 
> 1. uninstall explorer patcher
> 2. delete the registry keys: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher" "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher"
> 3. reboot
> 4. install the latest explorer patcher
> 5. verify you get a notification that symbols have been downloaded
> 6. restart windows explorer using either the task manager or in the properties window of explorer patcher
> 7. reboot
> 
> Then apply your settings again in explorer patcher to see if it fixes it.

**Comment by flavio-b-74** [comment]
Not working for me. It randomly crashes while opening file explorer... Win 11 22H2 (build SO 22621.2134)

**Comment by pyrates999** [comment]
Please try the new release of EP.

**Reply by flavio-b-74** [reply]
Nothing changed: uninstalled explorer patcher, deleted registry keys, rebooted, installed the last version (22621.1992.56.2), rebooted again.  It randomly crashes while opening file explorer.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"
